local-dev/secretstores.bicep

```diff
@@ -14,38 +14,27 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-@description('Information about what resource is calling this Recipe. Generated by Radius. For more information visit https://docs.radapp.dev/operations/custom-recipes/')
-param context object
-
 import kubernetes as kubernetes {
-  kubeConfig: ''
   namespace: context.runtime.kubernetes.namespace
+  kubeConfig: ''
 }
 
-var daprType = 'secretstores.kubernetes'
-var daprVersion = 'v1'
+param context object
 
-resource daprComponent 'dapr.io/Component@v1alpha1' = {
+resource dapr 'dapr.io/Component@v1alpha1' = {
   metadata: {
     name: context.resource.name
+    namespace: context.runtime.kubernetes.namespace
   }
   spec: {
-    type: daprType
-    version: daprVersion
+    type: 'secretstores.kubernetes'
+    version: 'v1'
     metadata: []
   }
 }
 
 output result object = {
-  // This workaround is needed because the deployment engine omits Kubernetes resources from its output.
-  // This allows Kubernetes resources to be cleaned up when the resource is deleted.
-  // Once this gap is addressed, users won't need to do this.
-  resources: [
-    '/planes/kubernetes/local/namespaces/${daprComponent.metadata.namespace}/providers/dapr.io/Component/${daprComponent.metadata.name}'
-  ]
   values: {
-    type: daprType
```

Comment:
Why are these being removed? Usually we want to pass these back to be available to the Radius resource incase they need to be referenced.

Reply:
I was testing with the same recipe that we use in functional tests. I added it back